teacher.rb

```diff
@@ -2,13 +2,15 @@
 
 class Teacher < Person
   attr_accessor :specialization
+  attr_reader :json_class
 
-  def initialize(specialization, age, name = 'unknown', parent_permission: true)
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true, json_class: 'Teacher')
     super(age, name, parent_permission: parent_permission)
     @specialization = specialization
+    @json_class = json_class
   end
 
   def can_use_services?
-    true
+    false
   end
```

Comment:
- Hey, :wave: Since a teacher can_use_service as they're above 18 years of age, why make it false instead of true? You should leave it as true as the user can automatically use the service as there's no age limitation.